src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -665,7 +665,7 @@ public void putEVSHistory(String code, String name, String operation, String ref
 		try {
 			History evsHistory = new History(userId.get(), code, name, operation, reference);
 			ByteArrayOutputStream b = writeRequestArgumentsIntoByteStream(evsHistory);
-			post(EVS_REC,
+			post(EVS_REC + "?projectid=" + projectId.get(),
```

Comment:
That could work although ya with the current API the passed in URL  would be partial since the post, delete, etc. methods prepend the base server address.